packages/calcite-components/src/components/tabs/resources.ts

```diff
@@ -1,3 +1,21 @@
+export const CSS = {
```

Comment:
I don't think this is the proper place for these moved constants since tabs does not use any of these. Please move and split up as needed per component.